orchestra/migrations/0083_update_staffbot_request_status_choices.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 2.2.9 on 2020-03-30 19:21
+
+from django.db import migrations, models
+
```

Comment:
(you can DM me this for privacy reasons) Manual testing
* create an up-to-date production snapshot
* Report on count of request statuses.
* Record the Available Tasks screen of a user with lots of available tasks (maybe both a CSS and a designer)
* Run the migration
* Look at the available tasks lists. They should look the same.
* Report on count of request statuses.

Reply:
Responded in DM.